keps/sig-node/3960-pod-lifecycle-sleep-action/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: "Pod lifecycle sleep action"
+kep-number: 3960
+authors:
+  - "@AxeZhan"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: 2023-04-22
+reviewers:
+  - "@thockin"
```

Comment:
I want to volunteer @bobbypage as reviewer too 😄 , but wait for him to confirm

Reply:
I think @bobbypage could be approver, too.  I'll server for API, but he's all over lifecycle stuff.

Reply:
@mrunalp Can you nominate an approver?

Reply:
I'm happy to be reviewer, thanks for tagging me!

Reply:
you can add bobby as reviewer and approver now @AxeZhan